assignment.js

```diff
@@ -113,6 +136,47 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const playerAChoice = "scissors"
+const playerBChoice = "paper"
+
+function whoWins(playerA, playerB) {
```

Comment:
@justinbudlow nice work covering all of the possible outcomes. 

The approach that you're using involves nested `if` statements and while it works, you will want to avoid nesting if possible. The reason is that nesting can add complexity to your code which makes things more challenging to debug. However, in this case the code is still easy to follow.